virtualmachine/vsphere/util.go

```diff
@@ -586,9 +595,21 @@ var cloneFromTemplate = func(vm *VM, dcMo *mo.Datacenter, usableDatastores []str
 	// to delete all the network cards and create VirtualDevice specs.
 	// For now only configure the datastore and the host.
 	relocateSpec := types.VirtualMachineRelocateSpec{
-		Pool:      &l.ResourcePool,
-		Host:      &l.Host,
-		Datastore: &dsMor,
+		Pool: &l.ResourcePool,
+	}
+
+	crMo, err := findClusterComputeResource(vm, dcMo, vm.Destination.DestinationName)
```

Comment:
IMO, if we are just interested in the drs enabled flag -- we could write a wrapper over  findClusterComputeResource which would return error and true/ flase